### ISSUE TITLE
[Assistive tech and device support] Focus state isn't visible. (11.09.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#27410]
#### VFS team name
Caregivers

#### VFS product name
Improve File Upload Error Messaging

#### Point of Contact/Reviewers
Josh Kim

---

### Platform Issue
Focus state isn't visible.

### Issue Details
Not within scope to error messaging in particular, but significantly impacts upload experience. Focus is lost on the single upload flow when a user uploads a document, cancels an upload, successfully uploads a document, and deletes a document.

### Link, screenshot or steps to recreate

https://user-images.githubusercontent.com/14154792/125811066-11fa893c-7bb7-43be-8fc6-cdb2c4f86dde.mov

### Supporting Guidance
[11.09](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.7_AA

---

### Platform Recommendation
When the user uploads a document, focus should be sent to the "cancel" button while the document is loading. When the user cancels an upload, focus should be circular and return to the upload button. When the user successfully uploads a document, focus should be sent to the document name input field. When the user deletes a document, focus should be circular and return to the upload button.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by joshkimux** [comment]
Hi @hdjustice (and I think y'all now have @davidakennedy right? ❤️ )-- we usually recommend resolving defect-2 issues within 1-2 sprints post-launch. Has there been any unexpected blockers on this? If y'all need support, feel free to reach out to the new CAIA team via #accessibility-help ❇️

**Comment by hdjustice** [comment]
Hi @joshkimux - Wow, this ticket is a bit overdue.  We will fit it into the next sprint.
